wic/src/main/webapp/resources/js/graphDisplay.js

```diff
@@ -1,98 +1,121 @@
 $(document).ready(function() {
-    var cy;
     $.ajax({
         url : url,
         type : "GET",
         success : function(result) {
+        	apiResult = result;
```

Comment:
is this a global variable that's then used in home.jsp?  This needs to be rewritten so that no global variables are used.